index.js

```diff
@@ -12,6 +12,10 @@ module.exports = function markdownItBidi(md) {
 
     const bidi = defaultRenderer => (tokens, idx, opts, env, self) => {
         const token = tokens[idx]
+        const prevToken = tokens[idx - 1]
+        if (token.type === 'th_open' && prevToken.type === "tr_open") {
+            return defaultRenderer(tokens, idx, opts, env, self)
+        }
```

Comment:
May you provide some more detail?

As far as I remember, tables were complex to deal with. I think it is better to merge any changes regarding tables after some reasonable number of tests. What do you think?

Reply:
That's the table:

![image](https://user-images.githubusercontent.com/42304709/151869989-ff2eae00-48ad-4c39-b6cd-e0141ea568b8.png)

What my change does is looking for \<th> that come just after \<tr>, returning before giving it the dir attribute.

> tables were complex to deal with

Not really, already tested and it works like a charm

> reasonable number of tests

Well, we should write them first

It's safe to merge, it just fixes a mistake I made, when I added `th_open`, `td_open` I forgot to add that check.